Proposal: using USWDS alerts for highlighting known issues on pages

adborden opened the pull request:
Use a [USWDS alert](https://designsystem.digital.gov/components/alert/) to highlight known issues for discussion on a particular page. This might also entice readers to help address the issues as they're reading handbook content.

- adds the alert component
- adds a "Needs Review" alert to the Responding to Public Disclosure Vulnerabilities page
- npm script to run prettier
- fix a rare issue with the sidenav injection


[Preview here](https://federalist-cf8341ad-ca07-488c-bd96-0738014c79ca.app.cloud.gov/preview/18f/handbook/feature/bug-bounty/responding-to-public-disclosure-vulnerabilities/)

![Screen Shot 2021-06-25 at 3 10 21 PM](https://user-images.githubusercontent.com/509703/124303991-f3720a00-db17-11eb-9bad-01b8709b6e1d.png)

Commit:
Add alert component as include

Commit:
Link to issues for discussion on vulnerabilities page

dylanirlbeck reviewed (dismissed):
I see some duplicate changes from #2586, so just be mindful of that (for future reviewers).

For the other stuff, it'd be nice if you included some screenshots and/or a GIF in the PR body! Either way, this LGTM, and could be a nice tool for getting more people to contribute.

Commit:
Fix sidenav injection

Make sure to inject after the first `p` tag that is a direct descendant of
`main`. Otherwise, the sidenav could be injected into a div container/component
that happens to contain a `p` tag.

Commit:
Run prettier locally

dylanirlbeck reviewed (approved):
LGTM! Only a couple comments and questions.

Commit:
Update README.md

Co-authored-by: Dylan Irlbeck  <dylanirlbeck@gmail.com>
